shared/src/main/scala/scalaclean/model/SharedModel.scala

```diff
@@ -141,6 +141,8 @@ package impl {
             PackagePathImpl(parent, encodedName)
         case _ if sym.isModule =>
           ObjectPathImpl(parent, name)
+        case _ if sym.isModuleOrModuleClass =>
```

Comment:
should be replacing line 142, not an additional case

Reply:
I meant to - I was hacking retesting afterward - will remove